### ISSUE TITLE
New Rule: line length

### ISSUE THREAD

**Issue by lonix1** [eslint/markdown#689]
### Rule details

Check line length

### What type of rule is this?

Warns about a potential problem

### Example code

```markdown
This is a very very very very very very very very very very very very very long not wrapped sentence.
Second sentence of of the paragraph,
third sentence of a paragraph
and the fourth one.
```

### Prior Art

```markdown
This exists in markdownlint's [MD013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md) rule.
```

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

Extra motivation notes [here](https://cirosantilli.com/markdown-style-guide/#line-wrapping).

**Comment by lumirlumir** [comment]
Thank you for opening an issue. However, under our policy, we don’t accept stylistic rules in ESLint core, and existing stylistic rules are feature-frozen.

Please read our policy on this here: https://eslint.org/docs/latest/use/core-concepts/glossary#stylistic-rule 

Thank you for understanding.
